main/src/cgeo/geocaching/settings/SettingsActivity.java

```diff
@@ -177,7 +151,7 @@ private void initPreferences() {
 
         for (final int k : new int[] {
                 R.string.pref_pass_vote, R.string.pref_signature,
-                R.string.pref_mapsource, R.string.pref_renderthemepath,
+                //R.string.pref_mapsource, R.string.pref_renderthemepath,
```

Comment:
Might be a leftover...

Reply:
It is indeed. I deleted the line and pushed the change.